src/Action/CancelAction.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\Payum\PayU\Action;
+
+use Answear\Payum\PayU\Enum\OrderStatus;
+use Answear\Payum\PayU\Model\Model;
+use Answear\Payum\PayU\Request\OrderRequestService;
+use Answear\Payum\PayU\Util\PaymentHelper;
+use Payum\Core\Action\ActionInterface;
+use Payum\Core\Exception\RequestNotSupportedException;
+use Payum\Core\Model\PaymentInterface;
+use Payum\Core\Request\Cancel;
+use Psr\Log\LoggerInterface;
+use Webmozart\Assert\Assert;
+
+class CancelAction implements ActionInterface
+{
+    private const FINAL_STATUSES = [
+        OrderStatus::Completed,
+        OrderStatus::Canceled,
+    ];
```

Comment:
move it to OrderStatus class,
or create a public method like `OrderStatus->isFinal()`